config/config_test.go

```diff
@@ -91,6 +91,10 @@ func TestReadConfig(t *testing.T) {
 			readValueFunc: func(c *Config) interface{} { return c.Extensions.TBTC.TBTCSystem },
 			expectedValue: "0xa4888eDD97A5a3A739B4E0807C71817c8a418273",
 		},
+		"Extensions.TBTC.LiquidationRecoveryTimeout": {
```

Comment:
I was split here between putting it in `TBTC` directly or `TBTC.Bitcoin`

Reply:
I like it better here 💯